io.github.amourspirit.LibrePythonista_PyEditor.yml

```diff
@@ -0,0 +1,62 @@
+command: cell_edit
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=home
+  - --filesystem=~/.var/app/org.libreoffice.LibreOffice:ro
+modules:
+  - name: cell_edit
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "librepythonista_python_editor==0.1.7" --no-build-isolation
+      - install -Dm755 cell_edit /app/bin/cell_edit
+      - install -Dm644 internal/ui/res/icon.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - install -Dm644 misc/$FLATPAK_ID.metainfo.xml $FLATPAK_DEST/share/metainfo/$FLATPAK_ID.metainfo.xml
+    sources:
+      - type: dir
+        path: .
+      - type: git
+        url: https://github.com/Amourspirit/LibrePythonista_PyEditor
+        tag: release-v0.1.0
+      - type: file
+        path: cell_edit
+      - type: file
+        url: https://files.pythonhosted.org/packages/7e/0a/a5260c758ff813acc6967344339aa7ba15f815575f4d49141685c4345d39/bottle-0.13.2-py2.py3-none-any.whl
+        sha256: 27569ab8d1332fbba3e400b3baab2227ab4efb4882ff147af05a7c00ed73409c
+      - type: file
+        url: https://files.pythonhosted.org/packages/c0/5a/9cac0c82afec3d09ccd97c8b6502d48f165f9124db81b4bcb90b4af974ee/jedi-0.19.2-py2.py3-none-any.whl
+        sha256: a8ef22bde8490f57fe5c7681a3c83cb58874daf72b4784de3cce5b6ef6edb5b9
+      - type: file
+        url: https://files.pythonhosted.org/packages/d2/97/d1d01417c2d39f673b52a7e331fb4f3af003fe33e608a92a450be9f4a5d6/librepythonista_python_editor-0.1.7-py3-none-any.whl
+        sha256: 24ce3d24ae67a942392b266f9f02f20d73fe7b7cdc42c215dabdb0a7d5897701
+      - type: file
+        url: https://files.pythonhosted.org/packages/c6/ac/dac4a63f978e4dcb3c6d3a78c4d8e0192a113d288502a1216950c41b1027/parso-0.8.4-py2.py3-none-any.whl
+        sha256: a418670a20291dacd2dddc80c377c5c3791378ee1e8d12bffc35420643d43f18
+      - type: file
+        url: https://files.pythonhosted.org/packages/f2/cf/77d3e19b7fabd03895caca7857ef51e4c409e0ca6b37ee6e9f7daa50b642/proxy_tools-0.1.0.tar.gz
+        sha256: ccb3751f529c047e2d8a58440d86b205303cf0fe8146f784d1cbcd94f0a28010
+      - type: file
+        url: https://files.pythonhosted.org/packages/b4/4f/fadb1eb352dbe8171dba983a7b8234af50e493b1a8aedb7fef902550019f/pywebview-5.3.2-py3-none-any.whl
+        sha256: 55f5a4bcfaa56738f2d6206977bd3d2864784249b0646c67e84f402ba1740000
+      - type: file
+        url: https://files.pythonhosted.org/packages/26/9f/ad63fc0248c5379346306f8668cda6e2e2e9c95e01216d2b8ffd9ff037d0/typing_extensions-4.12.2-py3-none-any.whl
+        sha256: 04e5ca0351e0f3f85c6853954072df659d0d13fac324d0072316b67d7794700d
+      
+  - name: pygobject
+    buildsystem: meson
+    config-opts:
+      - -Dpython=python3
+    sources:
+      - type: archive
+        url: "https://download.gnome.org/sources/pygobject/3.50/pygobject-3.50.0.tar.xz"
```

Comment:
it's in the runtime

Reply:
This is now corrected. PyObject is removed.